src/library/scala/collection/immutable/ChampHashMap.scala

```diff
@@ -58,6 +58,8 @@ final class ChampHashMap[K, +V] private[immutable] (private val rootNode: MapNod
 
   def get(key: K): Option[V] = rootNode.get(key, computeHash(key), 0)
 
+  override def getOrElse[V1 >: V](key: K, default: => V1): V1 = rootNode.getOrElse(key, computeHash(key), 0, default)
```

Comment:
Could you maybe add some tests for this override?

Reply:
added